plugins/generator-1.16.5/forge-1.16.5/procedures/get_command_parameter_blockstate.java.ftl

```diff
@@ -0,0 +1 @@
+(BlockStateArgument.getBlockState(cmdargs, "${field$param}").getState().getBlock())
```

Comment:
You call this one blockstate, but return the block

Reply:
Don't review this PR until the main PR is merged because they are a lot of things to change, fix and update. I will do everything when I will be able to test with the Blockly editor. It will be much easier.

Reply:
Ok